bundler/lib/bundler/templates/newgem/.circleci/config.yml.tt

```diff
@@ -0,0 +1,13 @@
+version: 2.1
+jobs:
+  build:
+    docker:
+      - image: ruby:<%= RUBY_VERSION %>
+    steps:
+      - checkout
+      - run:
+          name: Run the default task
+          command: |
+            gem install bundler -v <%= Bundler::Version %>
```

Comment:
```suggestion
            gem install bundler -v <%= Bundler::VERSION %>
```

The `Bundler::Version` constant doesn't exist.

This error actually makes `bundle gem <foo> --ci circle` crash. The current specs didn't catch it because some use `bundle "gem ...` instead of `bundle! "gem...`, and the former doesn't raise when errors happen.

Could you change those to always use the `bundle!` helper?

This is our test suite's fault in my opinion, and I plan to fix it: the "raising" version should be the default unless you explicitly opt-out.

Reply:
Ohhhh, so `bundle` should be used when we are testing for an error to be raised? Are there other situations where we should prefer to use `bundle`? I did a rough count of each used in the rubygems repo.

```
ag -Q 'bundle "' | wc -l
902
ag -Q 'bundle! "' | wc -l
386
```

I'll check `newgem_spec.rb` and make the appropriate changes.

Reply:
> Ohhhh, so bundle should be used when we are testing for an error to be raised? Are there other situations where we should prefer to use bundle?

Yep, I believe testing errors is the only use case, otherwise always use `bundle!`. My plan is to change `bundle` to start raising on error, and change the specs that test for errors to pass a `raise: false` option to opt out.